Codeinwp/neve
# fix: js error in customizer on nested accordion

### Summary


### Will affect the visual aspect of the product

NO

### Screenshots 

### Test instructions


1. Activate Neve
2. Go to **Customizer > Buttons > Primary Button**
3. When opening the **Hover** no error should be present inside the console.


Closes #3497.
